### PR TITLE
Licences, script arrays

### PR DESCRIPTION
**Pull request by Ash258**
1. [x] Wait for <https://github.com/lukesampson/scoop/pull/2377/> merge (Licence)
1. [x] Edit all script into array
1. [x] Check all comments, notes
    - [x] Make them array
1. [x] Edit licences
1. [x] Add app name to start of each app description
1. [ ] Squash merge

### CODE REVIEW

File: AutoClickers.json [hunk 1]
```diff
@@ -2,7 +2,9 @@
     "version": "2016.04.18",
     "description": "A full-fledged autoclicker with two modes of autoclicking, at your dynamic cursor location or at a prespecified location. The maximum amounts of clicked can also be set (or left as infinite).",
     "homepage": "https://sourceforge.net/projects/orphamielautoclicker/",
-    "license": "CC-BY-NC-2.0",
+    "license": {
```
Comment (rasa) [inline AutoClickers.json]: Typo

File: AirForSteam.json [hunk 2]
```diff
@@ -2,34 +2,39 @@
     "version": "2018-0406",
     "description": "An elegant and customizable skin for Steam.",
     "homepage": "http://airforsteam.com/",
-    "license": " CC-BY-NC-4.0",
+    "license": {
+        "identifier": "CC-BY-NC-4.0",
+        "url": "https://github.com/Outsetini/Air-for-Steam/blob/master/LICENSE.txt"
```
Comment (rasa) [inline AirForSteam.json]: If it’s an [SPDX](https://spdx.org/licenses/) license, I’m not including a URL, as it’s a standard license.

File: TODO/VMware.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
     "version": "14.1.2-8497320",
-    "description": "Industry standard for running multiple operating systems as virtual machines (VMs) on a single Linux or Windows PC.",
+    "description": "VMWare. Industry standard for running multiple operating systems as virtual machines (VMs) on a single Linux or Windows PC.",
```
Comment (rasa) [inline TODO/VMware.json]: I would prefer we add a `name` property. Agreed?

File: AutoClickers.json [hunk 4]
```diff
@@ -2,7 +2,9 @@
...
+        "indentifier": "CC-BY-NC-2.0"
```
Comment (rasa) [inline AutoClickers.json]: Should be identifier

File: IdleMaster.json [hunk 5]
```diff
@@ -2,7 +2,10 @@
     "version": "1.1.1",
     "description": "Get your Steam Trading Cards the Fast Way (Fast Mode Extension) ",
     "homepage": "http://www.steamidlemaster.com/",
-    "license": "GPL-2.0",
+    "license": {
```
Comment (rasa) [inline IdleMaster.json]: ` GPL-2.0` is deprecated. `GPL-2.0-only` is better

Comment (rasa) [reply IdleMaster.json]: Scroll to the end of https://spdx.org/licenses/

File: TODO/VMware.json (hunk 3 above)
Comment (Ash258) [reply TODO/VMware.json]: Not sure if would be usable for more people. I am doing this only for my custom search cmdlet for my end-users.

File: AirForSteam.json (hunk 2 above)
Comment (Ash258) [reply AirForSteam.json]: I am including URL whener it's possible and when licence file exists. I prefer to cite and document everything.

Comment (rasa) [reply AirForSteam.json]: I would prefer that, as well, but don't have the time to document the 800+ apps under scoop/extras/versions!

Comment (Ash258) [reply AirForSteam.json]: Feel your pain :/

**Pull request merged**